Display.cpp

```diff
@@ -0,0 +1,13 @@
+
+
+#include "Display.h"
+
+
+Display::Display(int w, int h) {
+    numPixelWidth = w; 
+    numPixelHeight = h; 
+}
+
+Display::~Display() { 
+    std::cout << "Display distroyed" << std::endl; 
+} 
```

Comment:
Curly braces go on their own line.
Follow the course coding standard.
- [ ] revise